indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -230,7 +263,7 @@ protected void innerMap(
                         .put(hashedDimensions)
                         .array()
           ).toBytesWritable(),
-          value
+          new BytesWritable(InputRowHelper.toBytes(inputRow, config.getSchema().getDataSchema().getAggregators()))
```

Comment:
there is no computation happening really to get them, its just traversing the object chain to get reference to aggregators, that is why I did not try to cache them.
but, I agree, caching them in a global variable would make it look cleaner. will do.